core/templates/dev/head/pages/exploration-editor-page/translation-tab/translation-tab.directive.ts

```diff
@@ -19,6 +19,8 @@
 require(
   'components/common-layout-directives/common-elements/' +
   'attribution-guide.directive.ts');
+require(
+  'components/forms/custom-forms-directives/audio-file-uploader.directive.ts');
```

Comment:
Where is this used?

Reply:
core/templates/dev/head/pages/exploration-editor-page/translation-tab/modal-templates/add-audio-translation-modal.template.html

Reply:
Ditto.